docs/specification/current.md

```diff
@@ -240,45 +240,66 @@ A prefetch token is a placeholder in a prefetch template that is *_replaced by i
 
 Prefetch tokens MUST be delimited by `{{` and `}}`, and MUST contain only the qualified path to a hook context field *_or one of the following user identifiers: `userPractitionerId`, 'userPractitioneRoleId', `userPatientId`, or `userRelatedPersonId`_*.
 
-Individual hooks specify which of their `context` fields can be used as prefetch tokens. Only root-level fields with a primitive value within the `context` object are eligible to be used as prefetch tokens. For example, `{{context.medication.id}}` is not a valid prefetch token because it attempts to access the `id` field of the `medication` field.
+Individual hooks specify which of their `context` fields can be used as prefetch tokens. Prefetch templates must start with one of these context elements and can then navigate to information available within the context element using [simple FHIRPath](https://hl7.org/fhir/fhirpath.html#simple).  For example, 
 
-##### Prefetch tokens identifying the user
-A prefetch template enables a CDS Service to learn more about the current user through a FHIR read, like so:
+There are two styles of prefetch expressions - read and search.  How the expressions are formatted and how the data is expected to come back varies for each.
+
+##### read prefetch
+
+A prefetch is determined to be a read if, after resolving the prefetch token, the result is a string with the form `[resource name]/{{[token]}}`.  For example:
 ```
 {
   "prefetch": {
-    "user": "{{context.userId}}"
+    "patient": "Patient/{{context.patient}}",
+    "user": "{{context.user}}"
   }
 }
 ```
-or though a FHIR search:
+
+`{{context.patient}}` resolves to a simple id.  `{{context.user}}` resolves to a resource type and simple id separated by a '/'.  In both cases, the result of resolving the prefetch template is a single string that matches a relative URL to a single resource instance.  That relative URL could be appended to the CDS Client's server base URL to form the URL for a valid read.
```

Comment:
```suggestion
`{{context.patient}}` resolves to a simple id (e.g., `123` for the Patient with id 123).  `{{context.user}}` resolves to a resource type and simple id separated by a '/' (e.g., `Patient/123` or `Practitioner/456`).  In both cases, the result of resolving the prefetch template is a single string that matches a relative URL to a single resource instance.  That relative URL could be appended to the CDS Client's server base URL to form the URL for a valid read.
```

Adding examples would help, but I'm still not clear what the *assumptions* here. Is this paragraph **defining** a new `context.patient` variable in context, and a new `context.user` variable?